pkg/api/v1/conversion_generated.go

```diff
@@ -26,7 +26,7 @@ import (
 	resource "k8s.io/kubernetes/pkg/api/resource"
 	unversioned "k8s.io/kubernetes/pkg/api/unversioned"
 	apiv1 "k8s.io/kubernetes/pkg/api/v1"
-	batchv1 "k8s.io/kubernetes/pkg/apis/batch/v1"
+	v1beta1 "k8s.io/kubernetes/pkg/apis/extensions/v1beta1"
```

Comment:
> @liggitt or @deads2k maybe you can shed some light on why this change is appearing and whether it is correct or not?

It's probably an import ordering problem that needs to be pinned by using a hard dependency.